src/sbt-test/sbtorgpolicies/custom-template-config/build.sbt

```diff
@@ -1,4 +1,4 @@
-import sbtorgpolicies.model._
+import sbtorgpolicies.model.{ApacheLicense, GitHubSettings}
```

Comment:
I still don't completely understand why the wildcard import didn't work. It might be related to the original `model.sbt` object in `model.scala` I had.